app/internal/event.py

```diff
@@ -1,12 +1,73 @@
+import logging
 import re
+from typing import List
 
+from email_validator import validate_email, EmailSyntaxError
 from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
 from starlette.status import HTTP_400_BAD_REQUEST
 
+from app.database.models import Event
+
 ZOOM_REGEX = re.compile(r'https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+')
 
 
-def validate_zoom_link(location):
+def raise_if_zoom_link_invalid(location):
     if ZOOM_REGEX.search(location) is None:
         raise HTTPException(status_code=HTTP_400_BAD_REQUEST,
                             detail="VC type with no valid zoom link")
+
+
+def get_invited_emails(invited_from_form: str):
+    invited_emails = []
+    for invited_email in invited_from_form.split(','):
+        invited_email = invited_email.strip()
+        try:
+            validate_email(invited_email, check_deliverability=False)
+            invited_emails.append(invited_email)
```

Comment:
Is this line might raise an exception? If not - please write it outside the try/except :)